k8s/upgrade/src/constant.rs

```diff
@@ -35,12 +35,21 @@ pub(crate) fn get_image_tag() -> String {
             let major = *(tag.first().unwrap_or(&""));
             let minor = *(tag.get(1).unwrap_or(&""));
             let patch = *(tag.get(2).unwrap_or(&""));
-            format!("v{major}-{minor}-{patch}")
+            format!("{major}-{minor}-{patch}")
```

Comment:
are we missing pre-release portion here?

Reply:
done